docs/changelogs/v2.9.0.md

```diff
@@ -0,0 +1,154 @@
+
+### Experimental features
+
+The following features are hidden or disabled by default as we don't guarantee stability. Learn more about experiments in [our documentation](https://coder.com/docs/v2/latest/contributing/feature-stages#experimental-features).
+
+- The `coder support` command generates a ZIP with deployment information, agent logs, and server config values for troubleshooting purposes. We will publish documentation on how it works (and un-hide the feature) in a future release (#12328) (@johnstcn)
+- Port sharing: Allow users to share ports running in their workspace with other Coder users (#11939) (#12119) (#12383) (@deansheather) (@f0ssel)
+  ![Port Sharing](https://raw.githubusercontent.com/coder/coder/main/docs/changelogs/images/sharable-ports.png)
+
+### Features
+
+#### Templates
+
+- `coder_script` can to add binaries and files to `PATH` with a bin directory (#12205) (@mafredri)
+- Add support for marking external auth providers as optional (#12021) (#12251) (@aslilac)
+  > This is done via a [Terraform property](https://registry.terraform.io/providers/coder/coder/latest/docs/data-sources/external_auth#optional) in the template.
+- Support custom order of agent metadata (#12066) (@mtojek)
+- Support `order` property of `coder_app` resource (#12077) (@mtojek)
+- Support `order` property of `coder_agent` (#12121) (@mtojek)
+- Support custom validation errors for number-typed parameters (#12224) (@mtojek)
+
+#### CLI
+
+- Support `--header` and `--header-command` in config-ssh (#10413) (@JoshVee)
+- Make URL optional for `coder login` (#10925) (#12466) (@elasticspoon)
+
+#### Dashboard
+
+- Support any file extension in the template editor (#12000) (@BrunoQuaresma)
+- Support creating templates from scratch (#12082) (@BrunoQuaresma)
+- Show previous agent scripts logs (#12233) (@BrunoQuaresma)
+- Show build logs on template creation (#12271) (@BrunoQuaresma)
+- Support zip archives for template files (#12323) (@BrunoQuaresma)
+- Show client errors in DERP Region health page (#12318) (@BrunoQuaresma)
+- Display error messages on ws and access url health pages (#12430)
+  (@BrunoQuaresma)
+- Warn users if they leave the template editor without publishing (#12406) (@BrunoQuaresma)
+- Add Confluence, NET, and MS Teams icons as static files (#12500) (#12512) (#12513) (@michaelbrewer)
+- Render markdown in template update messages (#12273) (@aslilac)
+
+#### Backend
+
+- Expose DERP server debug metrics (#12135) (@spikecurtis)
+- Add logSender for sending logs on agent v2 API (#12046) (@spikecurtis)
+- Clean up organization handling (#12142) (#12143) (#12146) (@Emyrk)
+- Change agent to use v2 API for reporting stats (#12024) (@spikecurtis)
+- Send log limit exceeded in response, not error (#12078) (@spikecurtis)
+- Add template activity_bump property (#11734) (@deansheather)
+- Add activity status and autostop reason to workspace overview (#11987) (@aslilac)
```

Comment:
Shouldn't this be in the dashboard section?

Reply:
Moved this to the dashboard and added a screenshot.

Reply:
yep!